### PR TITLE
tests: add precompile solidity e2e tests 

### PR DESCRIPTION
**Pull request by zsystm**
# Description

Closes: #111 

This PR adds extensive E2E test coverage for precompile modules and updates the local dev environment setup script.

## Pre-requisites PR (must be merged first)
- https://github.com/cosmos/evm/pull/267

## Summary of Changes
- Tests
  - Added **happy path test cases** for both query and tx precompiles.
    - Query tests validate correct return values and formatting.
    - Tx tests confirm proper event emission and state transitions.
  - Added **edge case** test case for staking precompile (revert scenario).
- `local_node.sh`
  - Reduced CometBFT timeouts to speed up block production during testing.
  - Generated deterministic validator key via `VAL_MNEMONIC`.
  - Enabled required precompiles (slashing, evidence).
  - Removed redundant `init-node.sh`.

### Happy Path
- [x] **bank**
  - [x] **query**
    - [x] `Balances`
    - [x] `TotalSupply`
    - [x] `SupplyOf`
- [x] **bech32**
  - [x] `HexToBech32`
  - [x] `Bech32ToHex`
- [x] **staking**
   - [x] **query**
     - [x] `Delegation`
     - [x] `UnbondingDelegation`
     - [x] `Validator`
     - [x] `Validators`
     - [x] `Redelegation`
     - [x] `Redelegations`
  - [x] **tx**
    - [x] `CreateValidator`
    - [x] `EditValidator`
    - [x] `Delegate`
    - [x] `Undelegate`
    - [x] `Redelegate`
    - [x] `CancelUnbondingDelegation`
- [x] **distribution**
  - [x] **query**
    - [x] `ValidatorDistributionInfo`
    - [x] `ValidatorOutstandingRewards`
    - [x] `ValidatorCommission`
    - [x] `ValidatorSlashes`
    - [x] `DelegationRewards`
    - [x] `DelegationTotalRewards`
    - [x] `DelegatorValidators`
    - [x] `DelegatorWithdrawAddress`
    - [x] `CommunityPool`
  - [x] **tx**
    - [x] `ClaimRewards`
    - [x] `SetWithdrawAddress`
    - [x] `WithdrawDelegatorReward`
    - [x] `WithdrawValidatorCommission`
    - [x] `FundCommunityPool`
    - [x] `DepositValidatorRewardsPool`
- [x] **erc20**
  - [x] **query**
    - [x] `Name`
    - [x] `Symbol`
    - [x] `Decimals`
    - [x] `TotalSupply`
    - [x] `BalanceOf`
    - [x] `Allowance`
  - [x] **tx**
    - [x] `Transfer`
    - [x] `TransferFrom`
- **evidence** (dependency: https://github.com/cosmos/evm/pull/271)
  - **query**
    - `Evidence`
    - `GetAllEvidence`
  - **tx**
    - `SubmitEvidence`
- [x] **gov**
  - [x] **query**
    - [x] `GetVotes`
    - [x] `GetVote`
    - [x] `GetDeposit`
    - [x] `GetDeposits`
    - [x] `GetTallyResult`
    - [x] `GetProposal`
    - [x] `GetProposals`
    - [x] `GetParams`
    - [x] `GetConstitution`
  - [x] **tx**
    - [x] `SubmitProposal`
    - [x] `Deposit`
    - [x] `CancelProposal`
    - [x] `Vote`
    - [x] `VoteWeighted`
- **ics20** (dependency: https://github.com/cosmos/evm/issues/262)
  - **query**
    - `Denom`
    - `Denoms`
    - `DenomHash`
  - **tx**
    - `Transfer` 
- [x] **p256**
  - [x] `p256`
- [x] **slashing**
  - [x] **query**
    - [x] `GetSigningInfo`
    - [x] `GetSigningInfos`
    - [x] `GetParams`
  - **tx**
    - `Unjail` (dependency: https://github.com/cosmos/evm/issues/261)
- [x] **werc20**
  - [x] **tx**
    - [x] `Deposit`
    - [x] `Withdraw`

### Edge Case
- [x] `tests/solidity/suites/precompiles/test/1_staking/0_edge_case_revert.js`

---

## Author Checklist

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.

I have...

- [x] tackled an existing issue or discussed with a team member
- [ ] left instructions on how to review the changes
- [x] targeted the `main` branch

## Reviewers Checklist

**All** items are required.
Please add a note if the item is not applicable
and please add your handle next to the items reviewed
if you only reviewed selected items.

I have...

- [ ] added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] confirmed all author checklist items have been addressed
- [ ] confirmed that this PR does not change production code
- [ ] reviewed content
- [ ] tested instructions (if applicable)
- [ ] confirmed all CI checks have passed

### CODE REVIEW

File: tests/solidity/suites/precompiles/test/1_staking/1_create_and_edit_validator.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        }
+    }
+
+    before(async () => {
+        [signer] = await hre.ethers.getSigners()
+
+        // Instantiate the StakingI precompile contract
+        staking = await hre.ethers.getContractAt('StakingI', STAKING_ADDRESS)
+        // Instantiate the Bech32I precompile contract for address conversion
+        bech32 = await hre.ethers.getContractAt('Bech32I', BECH32_ADDRESS)
```
Comment (cloudgray) [inline tests/solidity/suites/precompiles/test/1_staking/1_create_and_edit_validator.js]: Bech32 precmopile seems not to be used in this test.

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+                } catch {
+                    return false;
+                }
+            });
+            
+            const parsed = werc20.interface.parseLog(withdrawEvent);
+            console.log('Withdrawal event:', parsed.args);
+            expect(parsed.args.src).to.equal(signer.address);
+            expect(parsed.args.wad).to.equal(withdrawAmount);
+            console.log('No Withdrawal event found (expected for mock implementation)');
```
Comment (cloudgray) [inline tests/solidity/suites/precompiles/test/9_werc20/werc20.js]: I think this console.log is dead code. It will be unreachagle after expect assertion fails

File: tests/solidity/suites/precompiles/test/1_staking/0_edge_case_revert.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+const { expect } = require('chai');
+const hre = require('hardhat');
+
+describe('Staking – edge case revert test', function () {
+    const STAKING_PRECOMPILE_ADDRESS = '0x0000000000000000000000000000000000000800';
```
Comment (cloudgray) [inline tests/solidity/suites/precompiles/test/1_staking/0_edge_case_revert.js]: It is trivial but, how about create shared constants file for precompile addresses and gas limits and account addresses that are used?

File: tests/solidity/suites/precompiles/test/1_staking/1_create_and_edit_validator.js [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+            deposit,
+            {gasLimit: GAS_LIMIT}
+        )
+
+        // Wait for 2 confirmations and log the transaction hash
+        const receipt = await tx.wait(2)
+        console.log('Transaction hash:', receipt.hash)
+
+        // Find and parse the CreateValidator event from the transaction logs
+        const parsed = receipt.logs
```
Comment (cloudgray) [inline tests/solidity/suites/precompiles/test/1_staking/1_create_and_edit_validator.js]: This parsing logic is found in many places. how about make it helper function?

**Review by cloudgray — COMMENTED** [review]
LGTM! I left some trivial comments!

File: tests/solidity/suites/precompiles/test/1_staking/1_create_and_edit_validator.js (hunk 1 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/1_staking/1_create_and_edit_validator.js]: Applied in https://github.com/cosmos/evm/pull/253/commits/0693d094ecb279bff67aa910609755acb6c5f0a9

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js (hunk 2 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/9_werc20/werc20.js]: Applied in https://github.com/cosmos/evm/pull/253/commits/0693d094ecb279bff67aa910609755acb6c5f0a9

File: tests/solidity/suites/precompiles/test/1_staking/0_edge_case_revert.js (hunk 3 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/1_staking/0_edge_case_revert.js]: Applied in https://github.com/cosmos/evm/pull/253/commits/0693d094ecb279bff67aa910609755acb6c5f0a9

File: tests/solidity/suites/precompiles/test/1_staking/1_create_and_edit_validator.js (hunk 4 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/1_staking/1_create_and_edit_validator.js]: Applied in https://github.com/cosmos/evm/pull/253/commits/ff99df89cc52212118269cb2323e048d2b3b9b4b

File: tests/solidity/suites/precompiles/test/1_staking/2_delegate.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        const newShares = BigInt(delegateEvt.args.newShares.toString())
+        expect(newShares).to.be.equal(stakeShares)
+
+        // Query delegation after staking
+        const afterDelegation = await staking.delegation(signer.address, valBech32)
+        const afterBalance = BigInt(afterDelegation.balance.amount.toString())
+        console.log('Delegated amount after staking:', afterBalance.toString())
+
+        // ensure on-chain balance increased by at least stakeAmount
+        expect(afterBalance).to.be.at.least(
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/1_staking/2_delegate.js]: We should make this a strict equality check

File: tests/solidity/suites/precompiles/test/1_staking/4_redelegate.js [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+        const entryResp = entries[0]
+        // check RedelegationEntryResponse.redelegationEntry.initialBalance
+        expect(
+            BigInt(entryResp.redelegationEntry.initialBalance.toString())
+        ).to.equal(
+            afterR.entries[0].initialBalance,
+            'list entry initialBalance should match singular result'
+        )
+        // check RedelegationEntryResponse.balance
+        expect(
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/1_staking/4_redelegate.js:211]: let's check the validator delegations before and after redelegation

File: tests/solidity/suites/precompiles/test/2_distribution/2_withdraw_delegator_reward.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+        expect(evt.args.delegatorAddress).to.equal(signer.address);
+        expect(evt.args.validatorAddress).to.equal(valHex);
+        expect(evt.args.amount).to.be.a('bigint');
+        expect(evt.args.amount).to.be.greaterThan(currentReward.amount, 'Withdrawn amount should be greater than zero');
+
+        // Check state after withdrawal
+        const afterResult = await distribution.delegationRewards(signer.address, valBech32);
+        const afterReward = afterResult[0];
+        // afterReward should be less than currentReward
+        expect(afterReward.amount).to.be.lessThan(currentReward.amount, 'Rewards should be reduced')
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/2_distribution/2_withdraw_delegator_reward.js:74]: check user balances here as well

File: tests/solidity/suites/precompiles/test/2_distribution/3_claim_rewards.js [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+        const postRaw = await distribution.delegationTotalRewards(delegatorAddress)
+        const { delegationRewards: postDelegation, totalRewards: postTotal } = formatTotalRewards(postRaw)
+
+        console.log('Parsed delegationRewards after claim:', postDelegation)
+        console.log('Parsed totalRewards after claim:', postTotal)
+
+        // assert that totalRewards decreased by claimed amount (if only one denom)
+        const beforeTotal = totalRewards.reduce((acc, c) => acc + c.amount, 0n)
+        const afterTotal  = postTotal.reduce((acc, c) => acc + c.amount, 0n)
+        expect(afterTotal).to.lessThan(beforeTotal)
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/2_distribution/3_claim_rewards.js:86]: check balances for the user again

File: tests/solidity/suites/precompiles/test/2_distribution/5_fund_community_pool.js [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+        expect(evt.args.depositor).to.equal(signer.address);
+        expect(evt.args.denom).to.equal(coin.denom);
+        expect(evt.args.amount.toString()).to.equal(coin.amount.toString());
+
+        const afterPool = await distribution.communityPool();
+        const beforeAmt = beforePool.find(c => c.denom === coin.denom);
+        const afterAmt = afterPool.find(c => c.denom === coin.denom);
+        const start = beforeAmt ? BigInt(beforeAmt.amount.toString()) : 0n;
+        const end = afterAmt ? BigInt(afterAmt.amount.toString()) : 0n;
+        expect(end).to.gte(start + BigInt(coin.amount.toString()));
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/2_distribution/5_fund_community_pool.js]: check user balances here too

File: tests/solidity/suites/precompiles/test/2_distribution/6_deposit_validator_rewards_pool.js [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+        const evt = findEvent(receipt.logs, distribution.interface, 'DepositValidatorRewardsPool');
+        expect(evt, 'DepositValidatorRewardsPool event must be emitted').to.exist;
+        expect(evt.args.depositor).to.equal(signer.address);
+        expect(evt.args.validatorAddress).to.equal(VAL_HEX);
+        expect(evt.args.denom).to.equal(coin.denom);
+        expect(evt.args.amount.toString()).to.equal(coin.amount.toString());
+
+        const afterRewards = await distribution.validatorOutstandingRewards(VAL_BECH32);
+        const afterCoin = afterRewards.find(c => c.denom === coin.denom);
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/2_distribution/6_deposit_validator_rewards_pool.js:50]: let's check sender balances

File: tests/solidity/suites/precompiles/test/3_erc20/erc20.js [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+        const balance = await erc20.balanceOf(owner.address)
+        expect(balance).to.be.gt(0n)
+    })
+
+    it('should return zero allowance by default', async function () {
+        const allowance = await erc20.allowance(owner.address, spender.address)
+        expect(allowance).to.equal(0n)
+    })
+
+    it('should transfer tokens', async function () {
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/3_erc20/erc20.js:47]: let's check the events emitted here

File: tests/solidity/suites/precompiles/test/3_erc20/erc20.js [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+        // record pre-transfer balances and allowance
+        const prevBalance    = await erc20.balanceOf(recipient.address)
+        const prevAllowance  = await erc20.allowance(owner.address, spender.address)
+        console.log(`Pre-transfer balance of recipient: ${prevBalance}`)
+        console.log(`Pre-transfer allowance of spender: ${prevAllowance}`)
+
+        // spender pulls from owner → recipient
+        const tx = await erc20
+            .connect(spender)
+            .transferFrom(owner.address, recipient.address, amount, {gasLimit: GAS_LIMIT})
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/3_erc20/erc20.js:89]: ditto

File: tests/solidity/suites/precompiles/test/6_gov/gov.js [hunk 13]
```diff
@@ -0,0 +1,194 @@
...
+        const deposit = { denom: 'atest', amount: hre.ethers.parseEther('0.5') }
+
+        const depTx = await gov
+            .connect(signer)
+            .deposit(signer.address, globalProposalId, [deposit], { gasLimit: GAS_LIMIT })
+        const depRcpt = await depTx.wait(2)
+        const depEvt = findEvent(depRcpt.logs, gov.interface, 'Deposit')
+        expect(depEvt, 'Deposit event must be emitted').to.exist
+        expect(depEvt.args.proposalId).to.equal(globalProposalId)
+        expect(depEvt.args.depositor).to.equal(signer.address)
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/6_gov/gov.js:87]: before/after balances should be checked

File: tests/solidity/suites/precompiles/test/6_gov/gov.js [hunk 14]
```diff
@@ -0,0 +1,194 @@
...
+            .connect(signer)
+            .cancelProposal(signer.address, proposalIdToCancel, {gasLimit: GAS_LIMIT})
+        const cancelRcpt = await cancelTx.wait(2)
+        const cancelEvt = findEvent(cancelRcpt.logs, gov.interface, 'CancelProposal')
+
+        expect(cancelEvt, 'CancelProposal event must be emitted').to.exist
+        expect(cancelEvt.args.proposer).to.equal(signer.address)
+        expect(cancelEvt.args.proposalId).to.equal(proposalIdToCancel)
+
+        await expect(gov.getProposal(proposalIdToCancel)).to.be.reverted;
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/6_gov/gov.js:223]: check balances for cancellation (in the future, add multiple depositors that should receive their funds back)

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js [hunk 15]
```diff
@@ -0,0 +1,148 @@
...
+                const tx = await werc20.deposit({
+                    value: amount,
+                    gasLimit: GAS_LIMIT
+                });
+                const receipt = await tx.wait();
+                
+                console.log('Gas used for', hre.ethers.formatEther(amount), 'deposit:', receipt.gasUsed.toString());
+                
+                expect(receipt.status).to.equal(1);
+                expect(receipt.gasUsed).to.be.greaterThan(0);
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/9_werc20/werc20.js:81]: check balances on both the contract and sender

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js [hunk 16]
```diff
@@ -0,0 +1,148 @@
...
+                to: WERC20_ADDRESS,
+                value: depositAmount,
+                gasLimit: GAS_LIMIT
+            });
+            const receipt = await tx.wait();
+            
+            console.log('Fallback deposit gas used:', receipt.gasUsed.toString());
+            
+            expect(receipt.status).to.equal(1);
+            expect(receipt.gasUsed).to.be.greaterThan(0);
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/9_werc20/werc20.js:114]: ditto

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js [hunk 17]
```diff
@@ -0,0 +1,148 @@
...
+            
+            // Check that transaction was successful
+            expect(receipt.status).to.equal(1);
+            expect(receipt.gasUsed).to.be.greaterThan(0);
+            
+            // Look for Withdrawal event
+            const parsed = findEvent(receipt.logs, werc20.interface, 'Withdrawal');
+            console.log('Withdrawal event:', parsed.args);
+            expect(parsed.args.src).to.equal(signer.address);
+            expect(parsed.args.wad).to.equal(withdrawAmount);
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/9_werc20/werc20.js:158]: ditto

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js [hunk 18]
```diff
@@ -0,0 +1,148 @@
...
+                
+                const tx = await werc20.withdraw(amount, {
+                    gasLimit: GAS_LIMIT
+                });
+                const receipt = await tx.wait();
+                
+                console.log('Gas used for', hre.ethers.formatEther(amount), 'withdrawal:', receipt.gasUsed.toString());
+                
+                expect(receipt.status).to.equal(1);
+                expect(receipt.gasUsed).to.be.greaterThan(0);
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/9_werc20/werc20.js:188]: ditto

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js [hunk 19]
```diff
@@ -0,0 +1,148 @@
...
+            
+            const tx = await werc20.withdraw(zeroAmount, {
+                gasLimit: GAS_LIMIT
+            });
+            const receipt = await tx.wait();
+            
+            console.log('Gas used for zero withdrawal:', receipt.gasUsed.toString());
+            
+            expect(receipt.status).to.equal(1);
+            expect(receipt.gasUsed).to.be.greaterThan(0);
```
Comment (vladjdk) [inline tests/solidity/suites/precompiles/test/9_werc20/werc20.js:218]: ditto

File: tests/solidity/suites/precompiles/test/1_staking/4_redelegate.js (hunk 6 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/1_staking/4_redelegate.js:211]: Applied in https://github.com/cosmos/evm/pull/253/commits/109ab8587b594cd57e1600b3ae4d4e3407d2660a
Added checks to compare the delegation objects of both the src and dst validators before and after redelegation.

File: tests/solidity/suites/precompiles/test/2_distribution/2_withdraw_delegator_reward.js (hunk 7 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/2_distribution/2_withdraw_delegator_reward.js:74]: Applied in https://github.com/cosmos/evm/pull/253/commits/bd74eda94a5c8af413fe4ca82f38a81113b05bd3

File: tests/solidity/suites/precompiles/test/2_distribution/3_claim_rewards.js (hunk 8 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/2_distribution/3_claim_rewards.js:86]: Applied in https://github.com/cosmos/evm/pull/253/commits/b08b07a6dfa59a0a50d6cc516ddc9bb7d9865fe2

File: tests/solidity/suites/precompiles/test/1_staking/2_delegate.js (hunk 5 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/1_staking/2_delegate.js]: Applied in https://github.com/cosmos/evm/pull/253/commits/f814266eeac226279749600b0d3c0d52a28d05c5

File: tests/solidity/suites/precompiles/test/2_distribution/5_fund_community_pool.js (hunk 9 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/2_distribution/5_fund_community_pool.js]: Applied in https://github.com/cosmos/evm/pull/253/commits/8a17f8ff339f0a5d637d659bcfe693a61dcc270c

File: tests/solidity/suites/precompiles/test/2_distribution/6_deposit_validator_rewards_pool.js (hunk 10 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/2_distribution/6_deposit_validator_rewards_pool.js:50]: Applied in https://github.com/cosmos/evm/pull/253/commits/4f69fb65ea04f3180ed765db150699cf24fc957d

File: tests/solidity/suites/precompiles/test/3_erc20/erc20.js (hunk 11 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/3_erc20/erc20.js:47]: Applied in https://github.com/cosmos/evm/pull/253/commits/43d0962da5f6d43657d319a11500609404ac88a2

File: tests/solidity/suites/precompiles/test/3_erc20/erc20.js (hunk 12 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/3_erc20/erc20.js:89]: Applied in https://github.com/cosmos/evm/pull/253/commits/43d0962da5f6d43657d319a11500609404ac88a2

Note: This commit depends on #317 .
The issue was discovered while adding event assertions for transferFrom, and the fix has been applied via #317.

This test assumes that the fix is already in place, and confirms the correct behavior of the emitted event after transferFrom.

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js (hunk 19 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/9_werc20/werc20.js:218]: Applied in https://github.com/cosmos/evm/pull/253/commits/3b084be3626402f140e3df1ac51ff310b4548086

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js (hunk 18 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/9_werc20/werc20.js:188]: Applied in https://github.com/cosmos/evm/pull/253/commits/3b084be3626402f140e3df1ac51ff310b4548086

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js (hunk 17 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/9_werc20/werc20.js:158]: Applied in https://github.com/cosmos/evm/pull/253/commits/3b084be3626402f140e3df1ac51ff310b4548086

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js (hunk 16 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/9_werc20/werc20.js:114]: Applied in https://github.com/cosmos/evm/pull/253/commits/3b084be3626402f140e3df1ac51ff310b4548086

File: tests/solidity/suites/precompiles/test/9_werc20/werc20.js (hunk 15 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/9_werc20/werc20.js:81]: Applied in https://github.com/cosmos/evm/pull/253/commits/3b084be3626402f140e3df1ac51ff310b4548086

File: tests/solidity/suites/precompiles/test/6_gov/gov.js (hunk 13 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/6_gov/gov.js:87]: Applied in https://github.com/cosmos/evm/pull/253/commits/9320fa5c56d76280067f26a6d12e1ae13580319d

File: tests/solidity/suites/precompiles/test/6_gov/gov.js (hunk 14 above)
Comment (zsystm) [reply tests/solidity/suites/precompiles/test/6_gov/gov.js:223]: Applied in https://github.com/cosmos/evm/pull/253/commits/9320fa5c56d76280067f26a6d12e1ae13580319d
